thorlcr/activities/thactivityutil.cpp

```diff
@@ -374,36 +374,46 @@ void calcMetaInfoSize(ThorDataLinkMetaInfo &info, CThorInputArray &inputs)
         info.totalRowsMin = 0; // a good bet
 }
 
-void calcMetaInfoSize(ThorDataLinkMetaInfo &info, ThorDataLinkMetaInfo *infos, unsigned num)
+void calcMetaInfoSize(ThorDataLinkMetaInfo &info, const ThorDataLinkMetaInfo *infos, unsigned num)
 {
     if (!infos||(num<=1))
     {
         if (1 == num)
             info = infos[0];
+        else
+        {
+            info.fastThrough = true;
+            info.totalRowsMin = info.totalRowsMax = 0;
+        }
         return;
     }
     if (!info.unknownRowsOutput)
```

Comment:
unrelated.  This test is probably redundant/incorrect.  Was slightly confused that it was checked if > 1 files, but not if =1.